include/bluetooth/mesh/light_temp_srv.h

```diff
@@ -0,0 +1,169 @@
+	/** Pointer to new Temperature level to set. */
+	uint16_t *temp;
+	/**
+	 * Pointer to new Delta UV level to set.
+	 * @note NULL if not applicable.
+	 */
+	int16_t *delta_uv;
+	/** Transition time value in milliseconds. */
+	uint32_t time;
+	/** Message execution delay in milliseconds. */
+	uint32_t delay;
+};
+
+/** Light CTL Temperature Server state access handlers. */
+struct bt_mesh_light_temp_srv_handlers {
+	/** @brief Set the CTL Temperature state.
+	 *
+	 * @note This handler is mandatory.
+	 *
+	 * @param[in] srv Server to set the CTL Temperature state of.
+	 * @param[in] ctx Message context.
+	 * @param[in] set Parameters of the state change.
+	 * @param[out] rsp Response structure to be filled.
+	 */
+	void (*const set)(struct bt_mesh_light_temp_srv *srv,
+			  struct bt_mesh_msg_ctx *ctx,
+			  const struct bt_mesh_light_temp_cb_set *set,
+			  struct bt_mesh_light_temp_status *rsp);
+
+	/** @brief Get the CTL Temperature state.
+	 *
+	 * @note This handler is mandatory.
+	 *
+	 * @param[in] srv Server to get the CTL state of.
+	 * @param[in] ctx Message context.
+	 * @param[out] rsp Response structure to be filled.
+	 */
+	void (*const get)(struct bt_mesh_light_temp_srv *srv,
+			  struct bt_mesh_msg_ctx *ctx,
+			  struct bt_mesh_light_temp_status *rsp);
+};
+
+/**
+ * Light CTL Temperature Server instance. Should be initialized with
+ * @ref BT_MESH_LIGHT_TEMP_SRV_INIT.
+ */
+struct bt_mesh_light_temp_srv {
+	/** Light Level Server instance. */
+	struct bt_mesh_lvl_srv lvl;
+	/** Model entry. */
+	struct bt_mesh_model *model;
+	/** Publish parameters. */
+	struct bt_mesh_model_pub pub;
+	/** Transaction ID tracker for the set messages. */
+	struct bt_mesh_tid_ctx prev_transaction;
+	/** Acknowledged message tracking. */
+	struct bt_mesh_model_ack_ctx ack_ctx;
+	/** Handler function structure. */
+	const struct bt_mesh_light_temp_srv_handlers *handlers;
+	/** The last known Temperature Level. */
+	uint16_t temp_last;
+	/** The last known Delta UV Level. */
+	int16_t delta_uv_last;
+	/** Current Temperature range. */
+	struct bt_mesh_light_temp_range temp_range;
+};
+
+/** @brief Publish the current CTL Temperature status.
+ *
+ * Asynchronously publishes a CTL Temperature status message with the configured
+ * publish parameters.
+ *
+ * @note This API is only used publishing unprompted status messages. Response
+ * messages for get and set messages are handled internally.
+ *
+ * @param[in] srv Server instance to publish on.
+ * @param[in] ctx Message context to send with, or NULL to send with the
+ * default publish parameters.
+ * @param[in] status Current status.
+ *
+ * @retval 0 Successfully published a Generic OnOff Status message.
+ * @retval -ENOTSUP A message context was not provided and publishing is not
+ * supported.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ */
+int32_t
```

Comment:
return value type should be on the same line as the function, break the line in another way. (matching in the .c file).